src/runtime/components/prose/CodeTree.vue

```diff
@@ -149,15 +149,20 @@ onBeforeUpdate(() => rerenderCount.value++)
     <li
       v-for="(item, index) in items"
       :key="`${level}-${index}`"
+      role="treeitem"
```

Comment:
maybe we should have [`aria-selected`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-selected) here?

[Associated WAI-ARIA roles, states, and properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/treeitem_role#associated_wai-aria_roles_states_and_properties)